amps/ags/rm-community/rm-community-rest-api-explorer/src/main/webapp/definitions/gs-core-api.yaml

```diff
@@ -2092,6 +2212,248 @@ paths:
+        For example, the following JSON body adds two children:
+        ```JSON
+        [
+          {
+            "id":"a7c10f46-b85b-4de5-af1c-930056b736a7"
+          },
+          {
+            "id":"e0d79b71-be2b-4ce7-a846-a7c50cba20fb"
+          }
+        ]
+        ```
+
+        If you specify a list as input, then a paginated list rather than an entry is returned in the response body. For example:
+
+        ```JSON
+        {
+          "list": {
+            "pagination": {
+              "count": 2,
+              "hasMoreItems": false,
+              "totalItems": 2,
+              "skipCount": 0,
+              "maxItems": 100
+            },
+            "entries": [
+              {
+                "entry": {
+                  ...
+                }
+              },
+              {
+                "entry": {
+                  ...
+                }
+              }
+            ]
+          }
+        }
+        ```
+      operationId: addChildToHold
+      parameters:
+        - $ref: '#/parameters/holdIdParam'
+        - in: body
+          name: nodeId
+          description: The node id.
+          required: true
+          schema:
+            $ref: '#/definitions/HoldChild'
+      consumes:
+        - application/json
+      produces:
+        - application/json
+      responses:
+        '201':
+          description: Successful response
+          schema:
+            $ref: '#/definitions/HoldChildEntry'
+        '400':
+          description: |
+            Invalid parameter: **holdId** is not a valid format or **HoldChild** is not valid
+        '401':
+          description: Authentication failed
+        '403':
+          description: Current user does not have permission to add items to the hold
+        '404':
+          description: |
+            **holdId** does not exist
+        default:
+          description: Unexpected error
+          schema:
+            $ref: '#/definitions/Error'
+    get:
+      tags:
+        - holds
+      summary: Get children of a hold
+      description: |
+        Returns a list of children of a hold with id **holdId**.
+      operationId: getHoldChildren
+      parameters:
+        - $ref: '#/parameters/holdIdParam'
+        - $ref: '#/parameters/skipCountParam'
+        - $ref: '#/parameters/maxItemsParam'
+      consumes:
+        - application/json
+      produces:
+        - application/json
+      responses:
+        '200':
+          description: Successful response
+          schema:
+            $ref: '#/definitions/HoldChildPaging'
+        '401':
+          description: Authentication failed
+        '403':
+          description: Current user does not have permission to read **holdId**
+        '404':
+          description: "**holdId** does not exist"
+        default:
+          description: Unexpected error
+          schema:
+            $ref: '#/definitions/Error'
+  '/holds/{holdId}/items/{holdChildId}':
```

Comment:
Let's either standardise on `children` or `items`. I think either is a reasonable description.

Reply:
Good catch. Changed to `/holds/{holdId}/children/{holdChildId}`.